include/zenoh/api/publisher.hxx

```diff
@@ -41,7 +41,7 @@ class Publisher : public Owned<::z_owned_publisher_t> {
 
         /// @brief The encoding of the data to publish.
         std::optional<Encoding> encoding = {};
-        /// @brief the timestamp of this message.
+        /// @brief The timestamp of this message.
```

Comment:
above there were changes from capital to lowercase, here is the opposite change

Reply:
Yes, all the sentences start with upper case, argument descriptions start with lowercase (since grammatically they are not the sentence start)